ParabolInc/parabol
# feat: add "go to dashboard" or "sign in" button to "invitation link expired" screen

#6838

Authenticated user:

<img width="519" alt="image" src="https://user-images.githubusercontent.com/466991/192752308-b7882b0f-eefe-4061-ae5f-289e478f010d.png">

Non-authenticated user:

<img width="641" alt="image" src="https://user-images.githubusercontent.com/466991/192752379-5e00d593-5129-4124-afc7-ca8b675e4c2c.png">

How to test:

- Pick (or make) any expired team invite link from rethinkDB `actionDevelopment.MassInvitation` (see how to make link expired here: https://github.com/ParabolInc/parabol/pull/7153#issuecomment-1265332081)
- Open incognito window with no user logged in (anonymous)
- Open the link, see an error, see `Sign in` button and suggestion to Sign in to request an invite
- Login as another user, open the link, see `Request invite` button and `My dashboard` button
- Click on Request invite, see that invite request was sent successfully
